stable/prometheus-cloudwatch-exporter/templates/configmap.yaml

```diff
@@ -10,3 +10,7 @@ metadata:
 data:
   config.yml: |
 {{ printf .Values.config | indent 4 }}
+{{- if .Values.logging.enabled }}
```

Comment:
this ConfigMap will never be used by the deployment because it's missing a reference